site/class-1-bar/Bar/bar.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
You’re uploading files to `site/class-1-bar/Bar/` instead of `site/class-1-bar/sarahbenmessauoud/`!

Also: rename this file to `index.html`.